### DISCUSSION TITLE
Data available even after calling cache.clear()

### DISCUSSION THREAD

**Discussion by Arjun-sna** [vercel/swr#718] [Q&A]
I'm using `useSWR` in a component and rendering result. In that I have set `dedupingInterval` as 60 seconds. With that, a resource item in the previously fetch data can be updated in a separate component. Since `mutate` cannot be used in this case to revalidate the data, I tried clearing the cache.
But even after clearing the cache, the `useSWR` returns old data.

Basically the question is, is there a way to revalidate within the `dedupingInterval` time by clearing cache?

**Comment by huozhi** [comment]
@Arjun-sna  would you mind providing a minimal code example to explain what you want to achieve here? I saw you PR here https://github.com/vercel/swr/pull/720 , why would you like to only clean the cache in that case?
